src/plugins/check-unit-test/index.js

```diff
@@ -4,54 +4,59 @@
  * @author Gyandeep Singh
  */
 
+"use strict";
+
 const path = require("path");
 
 const TAG_REGEX = /^(?:Build|Chore|Docs|Upgrade):/;
 
 /**
  * Check if the commit message follow the guidelines
  * @param {string} message - commit message
- * @returns {boolean} True if its ok
+ * @returns {boolean} True if it's ok
  * @private
  */
-const isChoreTypePullRequest = (message) => TAG_REGEX.test(message);
+const isChoreTypePullRequest = TAG_REGEX.test.bind(TAG_REGEX);
 
 /**
  * Create the comment for the commit message issue
  * @param {string} username - user to tag on the comment
  * @returns {string} comment message
  * @private
  */
-const commentMessage = (username) => `
+function commentMessage(username) {
+    return `
 Hi @${username},
 
 It looks like you have made some code changes, but you have not added any unit tests. We require unit tests for all types of code changes.
 If this is a mistake, then please ignore this message.
 
 [//]: # (check-unit-test)
 `;
+}
 
 /**
  * Checks if any unit test files are present in the pull request
- * @param {Array<object>} files - collection of files inside the pull request
+ * @param {Array<Object>} files - collection of files inside the pull request
  * @param {string} repoUrl - url of the repo
  * @returns {boolean} True if atleast 1 test files is present
  * @private
  */
-const areUnitTestFilesPresent = (files, repoUrl) =>
-    files
-        .map((file) => file.blob_url)
-        .map((url) => path.dirname(url))
-        .map((url) => url.substring(repoUrl.length))
-        .some((url) => url.includes("test"));
+function areUnitTestFilesPresent(files, repoUrl) {
```

Comment:
:-1: .. arrow functions are so neat with implicit return...

Reply:
I generally don't mind using arrow functions either, but they're currently disallowed by `eslint-config-eslint` and I think it's good to be consistent across projects. I would be fine with changing it in `eslint-config-eslint` if you feel strongly about it, though.

Reply:
I personally like it but i do want to follow what the team likes. Now sometime you have to feel it to understand what you are voting for and that is why I choose the airbnb guide, since we were using way modern ES level vs are usual code base. This way other members of the team can get a feel for it.
I have felt the same way before, its only until you try/feel it before you can create a feeling/like/dislike for something.

Reply:
I see. I had originally thought that this project used `eslint-config-airbnb-base` because it wasn't always part of the `eslint` organization and was only moved into the organization later.

I'm a bit confused about what you mean by "getting a feel for it". Why would `eslint-config-airbnb-base` be more useful for that than `eslint-config-eslint`?

Reply:
> eslint-config-airbnb-base because it wasn't always part of the eslint organization

this is true because i developed this as an internal tool where i work before making it for eslint.

When i talk about feel i am not talking about `eslint-config-airbnb-base` vs `eslint-config-eslint` but rather airbnb uses more modern ES version techniques vs what eslint supports. Feel factors is more like beliveing something after you have actually tries it vs believing it bu just reading stuff (theory based)... its hard to explain i guess...